omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskFactoryWrapper.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.helper;
+
+import org.openmrs.scheduler.TaskDefinition;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.api.context.Context;
+
+import java.util.Collection;
+
+public class TaskFactoryWrapper {
+
+    private SchedulerService schedulerService = Context.getSchedulerService();
```

Comment:
We do not recommend setting services on declaration because they may not be setup yet by the time the class is loaded.

Reply:
So Shall I  create a service initialize function here and will call from TaskActionResource - Setup function to initiate the service?

Reply:
setup function sounds cool.

Reply:
So shall I initialize inside the constructor? 
```
    public TaskFactoryWrapper(){
        schedulerService = Context.getSchedulerService();
    }
```

Reply:
Better than on declaration. As long as this class is not a spring bean.

Reply:
I removed the declaration, and use  Context.getSchedulerService() in every place